### PR TITLE
Milestone 4: Style

### PR DESCRIPTION
**Pull request by carloshs1994**
# Hi there 👋🏻 

## Sass

The project was built using Sass. The only difference was that I needed to add an additional loader to React, but the result is overall the same (A CSS output). It has the same dynamic response, it's just that the output looks more organized.

### Changes implemented in "style-app" branch 

- ✅  My code follows the style guidelines of this project.
- ✅  Final design matches [Zeplin's](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design.

## Final notes

Is expected that this project meets the requirements. Please let me know if anything is left to be done or can be improved. Again, thanks for the review.

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
     <nav>
       <Link to="/">Books</Link>
       <Link to="/categories">Categories</Link>
     </nav>
-    <span>Log in</span>
+    <span>
+      <ImUser />
+    </span>
   </header>
 );
```
Comment (Emmyn5600) [inline src/components/Header.js:17]: - [x] Please try to meet with the design provided, I see you are almost there as well as closer to the design but there are few implementations remaining.

- [x] Original 
image

- [x] Yours
image

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @carloshs1994 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight :champagne:
- All linters are passing :heavy_check_mark:
- Good commit history :heavy_check_mark:
- Good PR description :heavy_check_mark:

## Required Changes :recycle:

- [x] Check the comment under the review too.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!:clap::clap::clap:

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Header.js (hunk 1 above)
Comment (carloshs1994) [reply src/components/Header.js:17]: # Done
Captura de Pantalla 2022-02-26 a la(s) 4 10 23

File: src/styles/index.scss [hunk 2]
```diff
@@ -1,72 +1,385 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Slab:wght@300;700&display=swap');
```
Comment (rustam-tolipov) [inline src/styles/index.scss:1]: - [x] [OPTIONAL] GOOD JOB MAN 👏🙌. I also love working with sass. I suggest trying a class naming concept called `BEM`. This makes waaaay easy to use the SASS 👍. Also, try this cheatsheet https://9elements.com/bem-cheat-sheet/

File: src/components/Book.js [hunk 3]
```diff
@@ -16,12 +16,47 @@ const Book = ({
   return (
```
Comment (rustam-tolipov) [inline src/components/Book.js:16]: - [x] [OPTIONAL] Also one thing in your functionality, please make a function that clears the input fields after a book is added, ok? 🍀😉 **HAPPY CODING**

**Review by rustam-tolipov — APPROVED** [review]
## Hi @carloshs1994 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/styles/index.scss (hunk 2 above)
Comment (carloshs1994) [reply src/styles/index.scss:1]: Great suggestion @Rustamxon7, I'll give it a try. Thanks for the help!

File: src/components/Book.js (hunk 3 above)
Comment (carloshs1994) [reply src/components/Book.js:16]: I'll do it! Thanks @Rustamxon7

**Pull request merged**